### DISCUSSION TITLE
开启翻译直到离开这个网站

### DISCUSSION THREAD

**Discussion by Lovecrafx** [immersive-translate/immersive-translate#1825] [Ideas]
### 新功能 | New Feature

1. 就像浏览器自带的翻译那样，开启翻译后，一直在这网站的任意子页面内都会触发翻译

### 为什么您想要添加这些功能 | Why you want to add these features

1. 虽然有“默认翻译英语网站”，这个功能还不够智能，有些网站是中文，带了一点英文就会触发翻译
2. 虽然有“默认翻译该网站”，但浏览的英文网站并不固定，有些网站希望翻译，有些不需要翻译

**Comment by theowenyoung** [comment]
可以在【进阶设置】 里设置临时开启翻译的网站时长。
